test/functional/ui/screen_basic_spec.lua

```diff
@@ -79,6 +87,31 @@ describe('Screen', function()
     end)
   end)
 
+  describe('startup', function()
+    it('default startup screen', function()
```

Comment:
IIUC this is the only test that needs laststatus=2. I think it would be clearer if the `set_session` were inlined inside this single `it()`. Or relocated to a separate describe() group.